src/modules/Dashboard/components/TakeNowModal/LabeledDropdown/LabeledUserDropdown.tsx

```diff
@@ -109,18 +175,32 @@ export const LabeledUserDropdown = ({
             fill={variables.palette[disabled ? 'on_surface_alfa38' : 'on_surface_variant']}
           />
         }
-        isOptionEqualToValue={(option, value) => value.id === option.id}
         fullWidth={true}
         value={value}
         onChange={(_e, newValue) => onChange(newValue)}
         clearOnBlur={false}
         handleHomeEndKeys
         loading={isSearching}
         loadingText={t('loadingEllipsis')}
-        noOptionsText={t('takeNowNotFound')}
+        noOptionsText={t('takeNow.modal.dropdown.notFound')}
         onInputChange={(_e, search) => debouncedSearchHandler(search)}
         {...rest}
       />
+      <RenderIf condition={shouldShowWarningMessage}>
+        <StyledFlexTopCenter
+          sx={{
```

Comment:
Missing `borderBottomLeftRadius` and `borderBottomRightRadius` when `shouldShowWarningMessage` is true

Reply:
Good call. Thanks!